pkgs/development/python-modules/pandoc/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, plumbum
+, ply
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "boisgera";
+    repo = pname;
+    rev = version;
+    hash = "sha256-53LCxthxFGiUV5go268e/VOOtk/H5x1KazoRoYuu+Q0=";
+  };
+
+  propagatedBuildInputs = [
+    plumbum
+    ply
+  ];
+
+  meta = with lib; {
+    description = "This project brings Pandoc's data model for markdown documents to Python.";
```

Besides the [period](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) at the end this is a bit opinionated.

```suggestion
    description = "Library Pandoc's data model for markdown documents";
```

---

Changed to the suggested description.